dev/com.ibm.ws.webcontainer.servlet.4.0_fat/test-applications/Servlet5TestApp.war/resources/jsp/JSPErrorsGenerator.jsp

```diff
@@ -0,0 +1,48 @@
+<%@ page  import="java.io.IOException" %> 
```

Comment:
Where did this JSP come from? Copyright? Seems to have more than necessary for the test at  first glance.